extensions/binding/org.eclipse.smarthome.binding.npt/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="npt"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ntp">
+		<label>Date</label>
+		<description>NTP Updated Date &amp; Time</description>
+
+		<channels>
+			<channel id="dateTime" typeId="dateTime-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="ntpServer" type="text">
+				<context>network_address</context>
+				<label>NTP Server</label>
+				<description>The NTP server name.</description>
+				<default>0.pool.ntp.org</default>
+				<required>true</required>

```

Comment:
Ok, I'll move it to the parameter.required level instead. (same for the other required one)
Yes, default is used to pre-populate the field